File: android/api-tests/src/test/java/com/revenuecat/apitests/java/EntitlementInfoMapperApiTests.java
package com.revenuecat.apitests.java.mappers;

import com.revenuecat.purchases.EntitlementInfo;
import com.revenuecat.purchases.hybridcommon.mappers.EntitlementInfoMapperKt;

import java.util.Map;

@SuppressWarnings("unused")
class EntitlementInfoMapperApiTests {
    private void checkMap(EntitlementInfo entitlementInfo) {
        Map<String, Object> map = EntitlementInfoMapperKt.map(entitlementInfo);
    }
}

File: android/api-tests/src/test/java/com/revenuecat/apitests/kotlin/mappers/EntitlementInfoMapperApiTests.kt
package com.revenuecat.apitests.kotlin.mappers

import com.revenuecat.purchases.EntitlementInfo
import com.revenuecat.purchases.hybridcommon.mappers.map

@Suppress("unused", "UNUSED_VARIABLE")
private class EntitlementInfoMapperApiTests {
    fun checkMap(entitlementInfo: EntitlementInfo) {
        val map: Map<String, Any?> = entitlementInfo.map()
    }
}
